src/View/Calendar.php

```diff
@@ -161,6 +161,15 @@ public static function renderTable( $atts ): string {
 		$print .= "</tbody></table>";
 		$print .= '</div>';
 
+		$print .= '<div id="cb-table-footnote">';
```

Comment:
@hansmorb können wir den Legenden-Block nicht als template-Part anlegen und in der Tabelle und dem Kalender identisch nutzen? Sonst haben wir den Teil ja doppelt gebaut und müssten bei Änderungen immer prüfen das alles konsistent nachzuziehen.

Reply:
War leichter gesagt als getan, weil der andere Shortcode auch erst in ein Template verschoben werden musste. Bitte prüfen!